src/utilities.rs

```diff
@@ -0,0 +1,59 @@
+use std::collections::HashSet;
+
+pub fn parse_config_value_set(config_value: String) -> HashSet<String> {
+    let mut value_set = HashSet::new();
+
+    let split_string = config_value.split(',');
+
+    for string in split_string {
+        value_set.insert(String::from(string));
+    }
+
+    value_set
```

Comment:
It looks like this function body is equivalent to

    config_value.split(',').map(String::from).collect()